production/helm/loki-stack/requirements.yaml

```diff
@@ -2,20 +2,20 @@ dependencies:
 - name: "loki"
   condition: loki.enabled
   repository: "file://../loki"
-  version: "^0.6.0"
+  version: "0.26.0"
```

Comment:
Why do we pin all those new dependencies ? This means we will need to update all the time. I think we should keep the `^`

Reply:
see pr info:

use fixed versions in loki-stack requirements to have real changes on loki-stack chart updates ( currently only loki-stack chart version is raised but you can't see changes of the dependencies in the commits)

Reply:
Sorry I don't understand, I'm wondering why we pin instead of supporting a range of versions. I don't want to have to update those dependencies each time, I was under the assumption that having `^0.26.0` would install at least 0.26.0 but higher if possible. I'm mostly concerned about internal charts like loki, promtail,fluent-bit.

Reply:
Imho this makes changes instransparent as you never know which version will be installed like using "latest" for a docker image.

Reply:
well `^` allow installation of non breaking change (everything except major version change) and I think it's easier for us to manage. I want to get bug fixes for my users when possible, I don't have time to check all other charts. (means “compatible with”)

Reply:
I don't think it makes sense to pin to a specific version pre 1.0. If anything, this should be updated to accept any version `< 1.0.0`. If I understand semver spec correctly the current `^0.26.0` will only accept `0.26.x`.

Reply:
I've added implicit versioning again.

Reply:
I don't see that.